platform_versioned_docs/version-24.1.0/enterprise/release_notes/enterprise_latest.mdx

```diff
@@ -4,18 +4,15 @@ Seqera Platform Enterprise version 24.1.0 introduces two new features: Data Expl
 
 ## Seqera Compute
 
-Seqera Compute is a fully optimized runtime designed to revolutionize scientific computing on Seqera Platform. Now available in private preview, Seqera Compute enables scientists to concentrate on their work by integrating compute, storage, credential management, and performance optimizations into a single, cohesive service. 
+Seqera Compute is a fully optimized runtime designed to revolutionize scientific computing on Seqera Platform. Now available in private preview, Seqera Compute enables scientists to concentrate on their work by integrating compute, storage, credential management, and performance optimizations into a single, cohesive service.
```

Comment:
IMO this sounds like marketing text right now, where can folks get more information about how to leverage it? Are there how-to docs elsewhere on the site this can link to?

Reply:
I agree, this seems it should be on our website as advertising for Seqera Compute, but seems out of place to be in technical documentation.

Reply:
Great feedback, thanks. I'm debating whether to include this tbh as it's not part of the release notes and was announced in BOS.

Reply:
Seqera Compute is not yet in public preview with this release. So it shouldn't be in these release notes. @mbosio85 can you confirm we should remove mention of Compute here for now?

Reply:
It definitely can be removed. Compute will not be available for enterprise deployment in 24.1